### DISCUSSION TITLE
ORGANIZACION LAUNCH-X-LATAM

### DISCUSSION THREAD

**Discussion by iguelAL** [Launch-X-Latam/MisionFrontEnd#61] [Q&A]
osea como? ¿El correo le da invitación a la organización launch-X-latam me tuvo que llegar al correo de innovación a mi correo personal?, ¿Tengo que vincular el correo de innovación a mi github o no es necesario? 
pd: ya estoy dentro de la organización, por eso estoy escribiendo aquí

**Comment by AlbertoCopado** [comment]
El correo de la invitación te tuvo que llegar al correo con el que creaste tu cuenta de GitHub.
Tu cuenta de GitHub la pudiste crear con tu correo personal, con el de Innovaccion ó con cualquier otra.
No es necesario vincular los correos, pero como recomendación, si creaste tu cuenta de GitHub con el correo de Innovaccion o de una institución educativa, deberías vincular tu correo personal para no perder acceso a tu cuenta de GitHub en caso de que por cualquier motivo las otras cuentas de correo fueran dadas de baja.
